src/scss/objects/_tables.scss

```
@@ -1,102 +1,105 @@
+      max-width: 100%;
+      margin-top: spacing(f);
+      margin-bottom: spacing(f);
+      border-spacing: 0;
+      background-color: $table-bgColor;
+      border: $table-border--width solid $table-border--color;
+      border-radius: dt.$radius-rounded-b;
 
-                &:first-child {
-                    border-left: 1px solid $table-innerBorder--color;
-                    border-radius: $radius-rounded-b 0 0;
-                }
-                &:last-child {
-                    border-right: 1px solid $table-innerBorder--color;
-                    border-radius: 0 $radius-rounded-b 0 0;
-                }
-            }
-            td {
-                &:last-child {
-                    border-right: none;
-                }
-            }
-        }
+      // Cells
+      & > thead,
+      & > tbody,
+      & > tfoot {
+          th,
+          td {
+              padding: $table-cell-padding;
+              line-height: v.$line-height-base;
+              vertical-align: top;
+              border-right: 1px solid $table-verticalBorder--color;
+          }
+          th {
+              text-align: left;
+              border-top: 1px solid $table-innerBorder--color;
 
-        // Bottom align for column headings
-        & > thead th,
-        & > thead td {
-            vertical-align: bottom;
-            border-bottom: 1px solid $table-innerBorder--color;
-        }
+              &:first-child {
+                  border-left: 1px solid $table-innerBorder--color;
+                  border-radius: dt.$radius-rounded-b 0 0;
+              }
+              &:last-child {
+                  border-right: 1px solid $table-innerBorder--color;
+                  border-radius: 0 dt.$radius-rounded-b 0 0;
+              }
+          }
+          td {
+              &:last-child {
+                  border-right: none;
+              }
+          }
+      }
 
-        // Account for multiple tbody instances
-        > tbody + tbody {
-            border-top: 1px solid $table-innerBorder--color;
-        }
+      // Bottom align for column headings
+      & > thead th,
+      & > thead td {
+          vertical-align: bottom;
+          border-bottom: 1px solid $table-innerBorder--color;
+      }
 
-        // Nesting
-        & table,
-        & .table {
-            background-color: $color-background-default;
-        }
+      // Account for multiple tbody instances
+      > tbody + tbody {
+          border-top: 1px solid $table-innerBorder--color;
+      }
 
-        &:not(.not-striped) {
-        tbody :nth-child(odd) th,
-        tbody :nth-child(odd) td {
-            background-color: $table-bgColor--accent;
-        }
-        }
-    }
- }
+      // Nesting
+      & table,
+      & .table {
+          background-color: dt.$color-background-default;
+      }
+
+      &:not(.not-striped) {
+      tbody :nth-child(odd) th,
+      tbody :nth-child(odd) td {
+          background-color: $table-bgColor--accent;
+      }
```

Nit: indentation

Also, could you do something like:
```suggestion
          tbody :nth-child(odd) {
              th,
              td {
                  background-color: $table-bgColor--accent;
              }
          }
```